read-fonts/src/tables/variations.rs

```diff
@@ -191,7 +196,26 @@ impl<'a> TupleVariationHeader<'a> {
         })
     }
 
+    /// Intermediate tuple records for this tuple variation table
+    /// — optional, determined by flags in the tupleIndex value.
+    #[inline(always)]
+    pub fn intermediate_tuples(&self) -> Option<(Tuple<'a>, Tuple<'a>)> {
+        self.tuple_index().intermediate_region().then(|| {
+            let start_range = self.shape.intermediate_start_tuple_byte_range();
+            let end_range = self.shape.intermediate_end_tuple_byte_range();
+            (
+                Tuple {
+                    values: self.data.read_array(start_range).unwrap(),
+                },
+                Tuple {
+                    values: self.data.read_array(end_range).unwrap(),
+                },
+            )
+        })
+    }
+
     /// Compute the actual length of this table in bytes
+    #[inline(always)]
```

Comment:
huh, where is this even used?

Reply:
This is used when parsing the tuple variation headers, each of which is variable size based on the header flags. Pretty sure you wrote this code :)